doc/api/http.md

```diff
@@ -1920,10 +1930,13 @@ changes:
 Node.js maintains several connections per server to make HTTP requests.
 This function allows one to transparently issue requests.
 
-`options` can be an object, a string, or a [`URL`][] object. If `options` is a
+`url` can be a string or a [`URL`][] object. If `url` is a
 string, it is automatically parsed with [`new URL()`][]. If it is a [`URL`][]
 object, it will be automatically converted to an ordinary `options` object.
 
+If both `url` and `options` are specified, the objects are merged, with the
+`options` properties taking precedence.
```

Comment:
including an example here would likely be helpful.

Reply:
I do have an example in the [test case](https://github.com/nodejs/node/pull/21616/commits/4365379b926eeafb09f9239b93c0913dcec46297#diff-83529a63e19144cdf716a756656e7e34R19), but it is a bit contrived.  While I am struggling to find a clear real world use case, I feel that the documentation would be incomplete without at least mentioning what would happen in the (however unlikely event) that there is overlap in the url and options.